src/style.css

```diff
@@ -0,0 +1,218 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-size: 62.5%;
+  background-color: rgb(240, 240, 240);
+  padding: 1rem;
+}
+
+.todo-box {
+  max-width: 70rem;
+  background-color: white;
+  margin: auto;
+  margin-top: 10%;
+  border-radius: 1rem;
+  overflow: hidden;
+  box-shadow: 0 0 30px rgb(144, 166, 238);
+}
+
+.header-holder {
+  background-image: url(./sky.webp);
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 1.5rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.header {
+  font-size: 3rem;
+  color: orangered;
+  text-align: center;
+  padding: 5rem 0;
+  padding-top: 2rem;
+}
+
+.form {
+  width: 100%;
+}
+
+.input-holder {
+  width: 100%;
+  padding: 1.5rem;
+  display: flex;
+  border-bottom: 1px solid lightblue;
+  transition: all 0.3s ease;
+}
+
+.bg-active {
+  background-color: rgb(215, 249, 250);
+}
+
+.input {
+  font-size: 2rem;
+  border: none;
+  outline: none;
+  width: 100%;
+}
+
+.checkbox {
+  font-size: 2rem;
+  padding: 1rem 1.2rem;
+  color: rgb(142, 186, 201);
+  transition: all 0.3s ease;
+  cursor: pointer;
+  align-self: center;
+  border-radius: 50%;
+}
+
+.checkbox:hover {
+  background-color: rgba(238, 216, 144, 0.295);
+}
+
+.icon {
+  font-size: 2rem;
+  padding: 1rem 1.2rem;
+  color: rgb(172, 73, 73);
+  transition: all 0.3s ease;
+  cursor: pointer;
+  align-self: flex-start;
+  display: none;
+}
+
+.enterIcon {
+  font-size: 2rem;
+  padding: 1rem 1.2rem;
+  color: rgb(111, 172, 192);
+  transition: all 0.3s ease;
+  cursor: pointer;
+}
+
+.enterIcon:hover {
+  color: black;
+}
+
+.visible {
+  display: block;
+}
+
+.edit {
+  font-size: 2rem;
+  padding: 1rem 1.8rem;
+  color: rgb(101, 167, 189);
+  transition: all 0.3s ease;
+  cursor: pointer;
+  align-self: flex-start;
+  border-radius: 50%;
+}
+
+.edit:hover {
+  color: black;
+}
```

Comment:
Line 115 has the same property with line 97 through 99, try and delete one and give both class the same property as intended to make your code clean.

.enterIcon:hover,
.edit:hover {
  color: black;
}